source/pug/views/fellowships.pug

```diff
@@ -7,7 +7,7 @@ block centerNav
 
 block content
   .container
-    .row.py-5
+    .row.pt-5
```

Comment:
Can we consistently use margins across these? This way we can take advantage of their collapsing nature and not worry about special markup for each case.